### PR TITLE
modify product

### PR DESCRIPTION
**Pull request by hoangtrungthong**
## Related Tickets:
- [#43652](https://edu-redmine.sun-asterisk.vn/issues/43652)

## WHAT this PR does?
change migration,model product

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by hoangtrungthong** [comment]
ready

File: database/migrations/2021_11_25_045209_create_products_table.php [hunk 1]
```diff
@@ -17,9 +17,9 @@ public function up()
             $table->id();
             $table->unsignedInteger('category_id');
             $table->string('name')->unique();
+            $table->string('slug');
```
Comment (huyenttm-0901) [inline database/migrations/2021_11_25_045209_create_products_table.php]: sửa cấu trúc bảng mà sửa file migration à em? (tat)
tạo file mới nha

File: database/migrations/2021_11_25_051133_create_product_attributes_table.php [hunk 2]
```diff
@@ -16,9 +16,10 @@ public function up()
         Schema::create('product_attributes', function (Blueprint $table) {
             $table->id();
             $table->unsignedInteger('product_id');
-            $table->json('color');
-            $table->json('ram');
-            $table->json('price');
+            $table->integer('color_id');
```
Comment (huyenttm-0901) [inline database/migrations/2021_11_25_051133_create_product_attributes_table.php]: tương tự

**Comment by hoangtrungthong** [comment]
ready

**Pull request merged**